frontend/components/SetupPage/index.tsx

```diff
@@ -85,8 +83,6 @@ export const Setup = () => {
         return <TransferFunds />;
       case SetupScreen.SetupBridgeOnboardingScreen:
         return <SetupBridgeOnboarding />;
-      case SetupScreen.SetupOnRamp:
-        return <SetupOnRamp />;
```

Comment:
is it correct that it's removed from here? while we are in onboarding, we should remain on Pages.Setup, not main